### DISCUSSION TITLE
Variable Transcription Times with Deepgram API

### DISCUSSION THREAD

**Discussion by Song-Junho97** [deepgram/community#567] [General help]
Our team is using the Deepgram API to transcribe **pre-recorded audio** with the **transcribe_file()** method, 
specifically utilizing the **'nova-2' model** option and **smart_format=true.** This is used for converting **English audio to text.** 

We call this API using **buffer data of the WAV type**. 
When transcribing short audio clips, sometimes **it takes less than 1.5 seconds** for the same audio, 
but other times **it takes as long as 17 seconds**, and in the worst case, **it has even taken 37 seconds**. 

**Why does this happen? 
Is there a way to resolve this issue?**

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by Song-Junho97** [reply]
` 
   
    source = {'buffer': buffer_data, 'mimetype': 'audio/wav'}
    options = PrerecordedOptions(smart_format=True, model="nova-2", language="en-US)

    response = deepgram.listen.prerecorded.v("1").transcribe_file(source, options, timeout=300)
`

**Reply by davidvonthenen** [reply]
Hi @Song-Junho97 

I dont know if it is related to your issue but the call to `transcribe_file` has an incorrect timeout parameter. It should be:
```
        response = deepgram.listen.prerecorded.v("1").transcribe_file(
            payload, options, timeout=httpx.Timeout(300.0, connect=10.0)
        )
```
